docs/docs/customization.md

```diff
@@ -50,6 +50,10 @@ For instance, building kyria firmware from a user `myUser`'s `zmk-config` folder
 west build -b nice_nano -- -DSHIELD=kyria_left -DZMK_CONFIG="C:/Users/myUser/Documents/Github/zmk-config/config"
 ```
 
+:::note
+You can install `west` using `pip3 install -U west`. Take a look [here](https://docs.zephyrproject.org/latest/guides/west/install.html) and [here](https://docs.zephyrproject.org/latest/getting_started/index.html#getting-started) for more info.
```

Comment:
Why not just point back to our ownr developer setup documentation that covers general setup of ZMK, including Zephyr dependencies?